tests/installation/install_service.pm

```diff
@@ -35,8 +35,22 @@ sub run {
     save_screenshot;
     assert_script_run 'systemctl status vsftpd --no-pager | grep active';
 
-    install_services($default_services) if (is_sle && !is_desktop && !is_sles4sap && !get_var('MEDIA_UPGRADE') && !get_var('ZDUP') && !get_var('INSTALLONLY'));
-    check_services($default_services)   if (is_sle && !is_desktop && !is_sles4sap && !get_var('MEDIA_UPGRADE') && !get_var('ZDUP') && !get_var('INSTALLONLY'));
+    install_services($default_services)
+      if is_sle
+      && !is_desktop
+      && !is_sles4sap
+      && !is_hyperv
+      && !get_var('MEDIA_UPGRADE')
+      && !get_var('ZDUP')
+      && !get_var('INSTALLONLY');
+    check_services($default_services)
+      if is_sle
+      && !is_desktop
+      && !is_sles4sap
+      && !is_hyperv
+      && !get_var('MEDIA_UPGRADE')
+      && !get_var('ZDUP')
+      && !get_var('INSTALLONLY');
```

Comment:
I guess it would be better to group the condition and only call both `install_services` and `check_services` within the same if-clause or did I misread it?

Reply:
Possibly. But I already proved I don't know the code enough to do such changes.